settings.js

```diff
@@ -12,30 +12,30 @@ var settings = {
   functionGlobalContext: {}, // enables global context
   httpNodeCors: {
     origin: '*',
-    methods: 'GET,PUT,POST,DELETE'
+    methods: 'GET,PUT,POST,DELETE',
```

Comment:
this is prettier error that adds unnecessary `,`.
While not major, can be easily fixed by adding `.prettierrc` copied from enebular

Reply:
I add .prettierrc file from enebular and save files again.